templates/identity-form.html

```diff
@@ -1,7 +1,7 @@
 
     % include "templates/sandbox-warning.html"
 
-    <p>{{ _("We need to know who you are because of financial regulations.") }}</p>
+    <p>{{ _("Liberapay needs more details about you to comply with financial regulations.") }}</p>
```

Comment:
This is clear about "compliance" being the detail to how financial regulations are over-held. Liberapay also already knows "who you are", in the sense that the friendly connection is made clear already.